### PR TITLE
Create read only user for grafana in database and configure Postgres data source

### PR DESCRIPTION
**Pull request by DaruZero**
Close https://github.com/united-manufacturing-hub/MgmtIssues/issues/983
Close https://github.com/united-manufacturing-hub/MgmtIssues/issues/984

### CODE REVIEW

**Comment by Scarjit** [comment]
Merge after #1672

File: deployment/united-manufacturing-hub/values.yaml [hunk 1]
```diff
@@ -934,7 +942,22 @@ grafana:
           # <bool> allow users to edit datasources from the UI.
           isDefault: true
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:943]: this should not be default anymore

File: deployment/united-manufacturing-hub/values.yaml [hunk 2]
```diff
@@ -934,7 +942,22 @@ grafana:
...
+            password: $GRAFANAREADER_PASSWORD
+          jsonData:
+            database: factoryinsight
+            sslmode: 'disable' # disable/require/verify-ca/verify-full
+            maxOpenConns: 100 # Grafana v5.4+
+            maxIdleConns: 100 # Grafana v5.4+
+            maxIdleConnsAuto: true # Grafana v9.5.1+
+            connMaxLifetime: 14400 # Grafana v5.4+
+            postgresVersion: 1308 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
+            timescaledb: true
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:960]: this data source should now be default :)

File: deployment/united-manufacturing-hub/values.yaml [hunk 3]
```diff
@@ -934,7 +942,22 @@ grafana:
...
+        # https://grafana.com/docs/grafana/latest/datasources/postgres/
+        - name: Postgres 
+          type: postgres
+          url: united-manufacturing-hub:5432
+          user: $GRAFANAREADER_USER
+          secureJsonData:
+            password: $GRAFANAREADER_PASSWORD
+          jsonData:
+            database: factoryinsight
+            sslmode: 'disable' # disable/require/verify-ca/verify-full
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:954]: sslmode=require

File: deployment/united-manufacturing-hub/values.yaml [hunk 4]
```diff
@@ -934,7 +942,22 @@ grafana:
...
+            maxOpenConns: 100 # Grafana v5.4+
+            maxIdleConns: 100 # Grafana v5.4+
+            maxIdleConnsAuto: true # Grafana v9.5.1+
+            connMaxLifetime: 14400 # Grafana v5.4+
+            postgresVersion: 1308 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:959]: can you double check with the postgres version that we are using? we should be at 14

File: deployment/united-manufacturing-hub/values.yaml [hunk 5]
```diff
@@ -934,7 +942,22 @@ grafana:
...
-        # <string, required> name of the datasource. Required
+        # https://grafana.com/docs/grafana/latest/datasources/postgres/
+        - name: Postgres 
+          type: postgres
+          url: united-manufacturing-hub:5432
+          user: $GRAFANAREADER_USER
+          secureJsonData:
+            password: $GRAFANAREADER_PASSWORD
+          jsonData:
+            database: factoryinsight
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:953]: database is now umh_v2

File: deployment/united-manufacturing-hub/values.yaml [hunk 6]
```diff
@@ -934,7 +942,22 @@ grafana:
           # <bool> allow users to edit datasources from the UI.
           isDefault: true
           editable: false
-        # <string, required> name of the datasource. Required
+        # https://grafana.com/docs/grafana/latest/datasources/postgres/
+        - name: Postgres
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:946]: Name: UMH TimescaleDB

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: deployment/united-manufacturing-hub/values.yaml (hunk 4 above)
Comment (DaruZero) [reply deployment/united-manufacturing-hub/values.yaml]: We are using 13.8

File: deployment/united-manufacturing-hub/values.yaml [hunk 7]
```diff
@@ -935,9 +946,25 @@ grafana:
...
+          secureJsonData:
+            password: $GRAFANAREADER_PASSWORD
+          jsonData:
+            database: umh_v2
+            sslmode: 'require' # disable/require/verify-ca/verify-full
+            maxOpenConns: 100 # Grafana v5.4+
+            maxIdleConns: 100 # Grafana v5.4+
+            maxIdleConnsAuto: true # Grafana v9.5.1+
+            connMaxLifetime: 14400 # Grafana v5.4+
+            postgresVersion: 13 # 903=9.3, 904=9.4, 905=9.5, 906=9.6, 1000=10
```
Comment (JeremyTheocharis) [inline deployment/united-manufacturing-hub/values.yaml:966]: 1300

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**